core/bin/contract-verifier/src/main.rs

```diff
@@ -24,9 +24,9 @@ async fn update_compiler_versions(connection_pool: &ConnectionPool<Core>) {
     let mut storage = connection_pool.connection().await.unwrap();
     let mut transaction = storage.start_transaction().await.unwrap();
 
-    let zksync_home = std::env::var("ZKSYNC_HOME").unwrap_or_else(|_| ".".into());
+    let zksync_home = locate_workspace().unwrap_or_else(|| ".".into());
```

Comment:
I think in this case it would be expected to provide `zksolc_path` / `solc_path` / ... as a part of the binary configuration (passed as env variables or whatever way we pass the config to binaries). Having it defined in terms of the workspace is an unnecessary and weird complication.

Reply:
i don't want to change current behaviour, but we can implement it cc: @perekopskiy

Reply:
I think it isn't worth an effort. Contract verifier will be reworked by devrel team soonish.